bundles/org.openhab.binding.neohub/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -148,6 +151,7 @@
 
 		<label>Heatmiser Contact Sensor</label>
 		<description>Heatmiser (wireless) Window or Door Contact</description>
+		<semantic-equipment-tag>Contact</semantic-equipment-tag>
```

Comment:
Note: Depends on discussion in other PR if this should be contact or not.

Reply:
Changed to `ContactSensor`